### PR TITLE
add alpine support in univers

### PR DESCRIPTION
**Pull request by TG1999**
Signed-off-by: Tushar Goel <tushar.goel.dav@gmail.com>

closes issue #37

### CODE REVIEW

**Review by pombredanne — CHANGES_REQUESTED** [review]
Thanks! I have a few nitpickings for your review!

File: tests/data/alpine_test.txt.ABOUT [hunk 1]
```diff
@@ -0,0 +1,9 @@
+about_resource: alpine_test.txt
+description: alpine version comparison tests
+license_expression: gpl-2.0
```
[the remark this answers is not in the document]
Comment (TG1999) [reply tests/data/alpine_test.txt.ABOUT:3]: We already have one `gpl-2.0.LICENSE` file in the data directory

File: src/univers/debian.py [hunk 2]
```diff
@@ -256,7 +256,9 @@ def compare_strings(version1, version2):
             logger.debug("Comparing non-digit prefixes %r and %r ..", p1, p2)
             for c1, c2 in zip_longest(p1, p2, fillvalue=""):
                 logger.debug(
-                    "Performing lexical comparison between characters %r and %r ..", c1, c2
```
[the remark this answers is not in the document]
Comment (TG1999) [reply src/univers/debian.py:259]: I ran just black, fixing it now

File: tests/test_maven_version.py [hunk 3]
```diff
@@ -240,7 +240,11 @@ def test_from_string(self):
 
         for test, expected in test_pairs:
             v = Version(test)
-            assert v._parsed == expected, "Version(%s) == %s, want %s" % (test, v._parsed, expected)
```
[the remark this answers is not in the document]
Comment (TG1999) [reply tests/test_maven_version.py:243]: I ran just black, fixing it now

File: src/univers/version_range.py [hunk 4]
```diff
@@ -721,6 +721,11 @@ class EbuildVersionRange(VersionRange):
     version_class = versions.GentooVersion
 
 
+class AlpineLinuxVersionRange(VersionRange):
+    scheme = "alpinelinux"
```
Comment (TG1999) [inline src/univers/version_range.py:725]: use scheme 'alpine'

Comment (TG1999) [inline src/univers/version_range.py:725]:
```suggestion
    scheme = "alpine"
```

File: src/univers/versions.py [hunk 5]
```diff
@@ -34,6 +34,21 @@ class InvalidVersion(ValueError):
     pass
 
 
+def valid_alpine_version(s):
```
Comment (TG1999) [inline src/univers/versions.py:37]:
```suggestion
def is_valid_alpine_version(s):
```

**Review by pombredanne — APPROVED** [review]
macOS is dead @Azure and we need to merge the latest skeleton, so ignore the failures.
This is looking great!
Thanks... merging!

**Pull request merged**